cupy/cudnn.pyx

```diff
@@ -1570,8 +1570,14 @@ cdef _get_dtype_of_tensor_descriptor(size_t desc):
 def batch_normalization_forward_training(
         core.ndarray x, core.ndarray gamma, core.ndarray beta,
         core.ndarray running_mean, core.ndarray running_var,
-        double eps, double decay,
+        mean, inv_std, double eps, double decay,
         bint is_for_conv2d, int cudnn_mode, bint debug):
+    # Usually supply None to mean and inv_std, which are left for backward
+    # compatibility. See cupy#2060 and cupy#2070.
+    if (mean is None and inv_std is not None or
+            inv_std is None and mean is not None):
```

Comment:
How about `if (mean is None) != (inv_std is None):`?

Reply:
Smart, fixed!